src/js/index.js

```diff
@@ -76,41 +73,71 @@ class FHCookieGuard {
         });
     }
 
-    /**
-     * @private
-     */
     _onAcceptCookiesClick() {
-        this._setCookie(1);
+        this._setCookie(this.options.cookieName, 1);
```

Comment:
kun je de functie niet hernoemen naar setCookieConsentValue en daar de waardes default in zetten.